db/migrate/20150130054629_add_category_to_assessments.rb

```diff
@@ -0,0 +1,18 @@
+class AddCategoryToAssessments < ActiveRecord::Migration
+  def change
+    add_column :assessments, :category_name, :string
+    
+    Assessment.all.each do |assessment|
+        sql = "SELECT name FROM assessment_categories WHERE id=" + assessment.category_id.to_s
+		records_array = ActiveRecord::Base.connection.execute(sql)
+	   	
+	   	records_array.each do |cat|
+    		assessment.category_name = cat[0]
+			assessment.save
+	    end
+    end
```

Comment:
No big deal, but indentation is off here.  What's wrong with adding the category name column, then running something like
```
Assessment.all.each do |assessment|
  assessment.category_name = assessment.category.name
  assessment.save!
end
```